.binder/environment.yml

```
@@ -0,0 +1,14 @@
+# based on https://github.com/jupyterlab-contrib/jupyterlab-vim/blob/master/binder/environment.yml
+name: burnman-binder-demos
+
+channels:
+  - conda-forge
+
+dependencies:
+  # runtime dependencies
+  - python >=3.8,<3.10.0a0
+  - jupyterlab >=3,<4.0.0a0
+  - pip
+  - ipympl
+  - ipywidgets
+  - appmode
```

You can also install mpl-interactions here rather than in postBuild:

```suggestion
  - appmode
  - pip:
    - mpl-interactions
```

I'm not sure but I'd guess that this will help with the caching on binder

---

Thank you again :) This is more rapid interaction than there's been for a while on this repo...